cmd/eksctl/main.go

```diff
@@ -3,9 +3,13 @@ package main
 import (
 	"fmt"
 	"os"
+	"runtime"
+	"strconv"
+	"strings"
 
 	"github.com/kris-nova/logger"
 	"github.com/spf13/cobra"
+	k8sruntime "k8s.io/apimachinery/pkg/util/runtime"
```

Comment:
We don't use `k8s` as import prefix, import it as `utilsruntime`, per Kubernetes convention.